client/src/main/kotlin/tech/figure/classification/asset/client/client/impl/DefaultACExecutor.kt

```diff
@@ -33,24 +33,10 @@ class DefaultACExecutor(
     private val pbClient: PbClient,
     private val querier: ACQuerier,
 ) : ACExecutor {
-    /**
-     * Automatically derives the message to onboard the asset for the specified verifier,
-     * if it exists for the provided asset type.
-     */
     override fun <T> generateOnboardAssetMsg(
         execute: OnboardAssetExecute<T>,
         signerAddress: String,
-    ): MsgExecuteContract = querier
```

Comment:
This upfront query was only necessary because we needed to use the verifier from the contract to determine how much "funds" to put in the msg.  Now that that is made irrelevant thanks to Provenance MsgFees, this will work without any magic at all.